models/devicesim-1.0.x/Makefile

```diff
@@ -65,25 +64,27 @@ test: mod-update # @HELP Run the unit tests
 
 repo-tag:
 ifeq ($(IS_RELEASED_VERSION), true)
-ifneq ("$(HAS_CHANGED)", "0 ")
-	NEW_VERSION=${VERSION} ../../build/build-tools/tag-collision-reject
```

Comment:
This is not going to work on Jenkins

You will need `build/build-tools/tag-collision-reject`